src/chai/RajaExecutionSpacePlugin.cpp

```diff
@@ -25,13 +25,25 @@ RajaExecutionSpacePlugin::preCapture(const RAJA::util::PluginContext& p)
 
   switch (p.platform) {
     case RAJA::Platform::host:
-      m_arraymanager->setExecutionSpace(chai::CPU); break;
+#if defined(CHAI_ENABLE_GPU_SIMULATION_MODE)
```

Comment:
would this be cleaner if the logic got pushed into setExecutionSpace?

Reply:
That's a good idea.

Reply:
Done.